cmsplugin_offices_ai/models/personnel.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -:-
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import gettext_lazy as _
+from filer.fields.image import FilerImageField
+
+from cmsplugin_offices_ai.models import Department, Office
+
+
+class PersonQuerySet(models.QuerySet):
+
+    def in_order(self):
+        """
+        Orders queryset by `ordering` field value
+        """
+        return self.order_by('ordering')
+
+    def public(self):
+        """
+        Exclude personnel that are not supposed to be shown in lists from queryset.
+        """
+        return self.filter(show_in_list=True).prefetch_related('person_contact_information')
+
+
+@python_2_unicode_compatible
+class Person(models.Model):
+    name = models.CharField(
+        verbose_name=_('name'),
+        max_length=128,
+        blank=False,
+    )
+    title = models.CharField(
+        verbose_name=_('title'),
+        max_length=128,
+        blank=True,
+    )
+    description = models.TextField(
+        verbose_name=_('description'),
+        max_length=512,
+        blank=True,
+    )
+    image = FilerImageField(
+        verbose_name=_('image'),
+        related_name='+',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+    )
+
+    office_override = models.ForeignKey(
+        Office,
+        verbose_name=_('office'),
+        related_name='personnel_in_office',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        help_text=_('Overrides the office the person is assumed to belong to via selected department.')
+    )
+    department = models.ForeignKey(
+        Department,
+        verbose_name=_('department'),
+        related_name='personnel_in_department',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+    )
+
+    # Display options
+    show_in_list = models.BooleanField(
```

Comment:
It flags the person visible in plugins that list personnel. This allows person to be hidden from department list plugin personnel lists for example.